src/handlers/predict.py

```diff
@@ -3,9 +3,13 @@
 
 
 def model_post_handler(request: PredictionRequestPydantic):
-    model = model_decoder.decode(request.rawModel[0])
-    data_entry_all = json_to_predreq.decode(request)
-    _ = model(data_entry_all)
+    model = model_decoder.decode(request.rawModel[0]) 
+    if len(request.dataset['features']) == 1:
+        data_entry_all = json_to_predreq.decode(request)
+        _ = model(data_entry_all)
+    else:
+        Smiles_input, data_entry_all = json_to_predreq.decode_with_external(request)
+        _ = model(Smiles_input, data_entry_all)
```

Comment:
This is too specific to the external descriptors example. The inference repository has to be as generic as possible, so it can be used by any generic python model, but here we're inputting `Smiles_input`. 

I'd propose to make this in a way that: 
- jaqpotpy turns the model in a generic way so that each model can be interpreted by the inference package here
- jaqpot-inference receives a generic request and has no logic, it just runs every model in the same way

so the whole logic is in the jaqpotpy library